docs/ybm_cluster_encryption_update-state.md

```diff
@@ -0,0 +1,37 @@
+## ybm cluster encryption update-state
+
+Update Encryption at Rest (EaR) state for a cluster
+
+### Synopsis
+
+Update Encryption at Rest (EaR) state for a cluster
+
+```
+ybm cluster encryption update-state [flags]
+```
+
+### Options
+
+```
+      --enable   [REQUIRED] Whether to enable or disable EAR (default true)
```

Comment:
Can we add another option disable it would be more intuitive to use than enable=true, enable/disable need not have additional options

Reply:
Do you mean adding two flags - enable/disable?

Reply:
Yes

Reply:
Yes, that will be a better user interface.
will modify to support both enable and disable.

`./ybm cluster encryption update-state --cluster-name test-cluster-aws --enable`
`./ybm cluster encryption update-state --cluster-name test-cluster-aws --disable`